.golangci.yml

```diff
@@ -7,28 +7,31 @@ issues:
   exclude-rules:
     - linters: [gosec]
       path: "test/e2e/*"
+
+linters-settings:
+  govet:
+    enable=fieldalignment: true
+
 linters:
   disable-all: true
   enable:
     - deadcode
     - dupl
     - errcheck
+    - exportloopref
     - goconst
     - gocyclo
     - gofmt
     - goimports
-    - golint
     - gosec
     - gosimple
     - govet
     - ineffassign
-    - interfacer
     - lll
-    - maligned
```

Comment:
WARN [runner] The linter 'maligned' is deprecated (since v1.38.0) due to: The repository of the linter has been archived by the owner.  Replaced by govet 'fieldalignment'.